src/osd/OSD.cc

```diff
@@ -6926,18 +6926,21 @@ void OSD::_committed_osd_maps(epoch_t first, epoch_t last, MOSDMap *m)
 	if (r != 0) {
 	  do_shutdown = true;  // FIXME: do_restart?
           network_error = true;
+          dout(0) << __func__ << " marked down: rebind failed" << dendl;
```

Comment:
for qa, I think developers could know the messenger type from job describe. for user, it should definitely be one type defined in ceph.conf